src/main/java/com/yana/internship/repository/ApartmentSpecification.java

```diff
@@ -0,0 +1,40 @@
+package com.yana.internship.repository;
+
+import com.yana.internship.entity.Address;
+import com.yana.internship.entity.Apartment;
+import com.yana.internship.entity.Country;
+import com.yana.internship.entity.Order;
+import org.springframework.data.jpa.domain.Specification;
+
+import javax.persistence.criteria.Join;
+import javax.persistence.criteria.JoinType;
+import javax.persistence.criteria.Predicate;
+import java.util.Date;
+
+public class ApartmentSpecification {
```

Comment:
Should be also Bean. 
Please rename this class. SpecificationBuilder or something like this